content/html/concepts/elements/terms/textarea/textarea.md

```diff
@@ -0,0 +1,60 @@
+---
+Title: '<textarea>'
+Description: 'An HTML element for multi-line plain-text input.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Elements'
+  - 'Input'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`<textarea>`** element defines a box for multi-line plain-text input.
+
+## Syntax
+
+```
+
+<textarea>
+  <!-- content here -->
+</textarea>
+
+```
+
+Shown above is the syntax for plain `<textarea>`. All attributes for this field are optional.
+
+### Attributes
+
+Commonly used attributes are shown in the table below.
+
+    | Attribute     | Data Type     | Description  |
+    | ------------- |-------------  | -----|
+    | `id`          | String        |   Links to `<label>` element |
+    | `name`        | String        |   Name of the table |
+    | `rows`        | Integer       |   Define the initial rows number|
+    | `cols`        | Integer       |   Define the initial columns number |
+
+When not given, the default value is 20 for `cols` and 2 for `rows`.
+
+## Example
+
+Following example demonstrates `<textarea>` with several attributes defined and associated `<label>` element. It can be used for comment section.
+
+```html
+
+<html>
+  <head> </head>
+  <body>
+    <label for="textbox">Share your thoughts:</label>
+    <textarea id="textbox" name="textbox"
+      rows="21" cols="37">
+        Okay, here's the situation...
+    </textarea>
+  </body>
+</html>
+
```

Comment:
In addition to removing the empty lines, I think it would be good to include an image for what will be displayed.

Use this format for an image and upload the image into the `docs/media/` folder with the proper naming. For example, if we want to name the file `html-elements-textarea.png`, then we would write
 
```md
This will display:

![HTML Elements Textarea label](https://raw.githubusercontent.com/Codecademy/docs/main/media/html-elements-textarea.png)
```

<img width="392" alt="Screen Shot 2022-12-30 at 8 01 21 AM" src="https://user-images.githubusercontent.com/72277593/210072814-1ac0ad61-16b0-47e5-bb96-826eac74382c.png">